src/lib/components/consignments/setup/info.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import Button from "../../buttons/inverted_button"
+import Welcome from "./welcome"
+
+import {
```

Comment:
Things have changed old man:

![screen shot 2017-05-19 at 16 55 41](https://cloud.githubusercontent.com/assets/49038/26256180/053b8722-3cb4-11e7-904b-21bed4abff8b.png)

Looks like it catches them now 👍 - will be fixed in an upcoming PR